[BE-3179]: audit permission changes docs added (#443)
appcircleio/appcircle-docusaurus: docs/workflows/ios-specific-workflow-steps/auidit-permission-change.md

---
title: Audit Permission Changes
description: Track and audit permission changes effectively. Ensure security and compliance with our guide on monitoring access rights alterations.
tags: [cocoapods, install, workflow, step]
sidebar_position: 6
---

import Screenshot from '@site/src/components/Screenshot';

# Audit Permission Changes

> With this component, you can capture and compare permission changes in your iOS projects. 

Reviewer: ```suggestion
This component captures and compares permission changes in your iOS projects.
```

> This check is conducted using a reference branch. First, you must run the pipeline containing this component on the branch you wish to use as a reference. Subsequently, when running on the branch you want to check, the control will be conducted relative to this selected reference branch. When there is a change in the permissions within the project, Appcircle automatically stops the pipeline to help ensure that a potentially significant permission change does not go unnoticed.

Reviewer: ```suggestion
The system conducts this check using a reference branch. First, run the pipeline containing this component on the branch you intend to use as a reference. Subsequently, when you run it on the branch you want to check, the system performs the control relative to this selected reference branch. When it detects a change in the project's permissions, Appcircle automatically stops the pipeline to ensure that no significant permission change goes unnoticed.
```

### Prerequisites

Before running the **Audit Permission Changes** step, you must complete certain prerequisites, as detailed in the table below:

| Prerequisite Workflow Step                      | Description                                     |
|-------------------------------------------------|-------------------------------------------------|

> | [Git Clone](/workflows/common-workflow-steps/git-clone) | The repo needs to be cloned in order to start the Audit Permission Changes process. After the clone, CocoaPods will be installed. After this step works, the variable `AC_REPOSITORY_DIR` will be created. |

Reviewer: ```suggestion
| [Git Clone](/workflows/common-workflow-steps/git-clone) | You need to clone the repository to start the Audit Permission Changes process. After cloning, the system installs CocoaPods. After this step works, the variable `AC_REPOSITORY_DIR` will be created. |
```
